### PR TITLE
Move examples from mesa repo to mesa-examples repo. 

### PR DESCRIPTION
**Pull request by jackiekazil**

### CODE REVIEW

File: tests/test_examples.py [hunk 1]
```diff
@@ -9,6 +9,9 @@ def classcase(name):
     return "".join(x.capitalize() for x in name.replace("-", "_").split("_"))
 
 
+@unittest.skip(
```
Comment (jackiekazil) [inline tests/test_examples.py:12]: This test is dependent on examples folder being here, so we are skipping until we discuss testing strategy.

File: tests/test_main.py [hunk 2]
```diff
@@ -19,6 +19,9 @@ def setUp(self):
     def tearDown(self):
         sys.path[:] = self.old_sys_path
 
+    @unittest.skip(
```
Comment (jackiekazil) [inline tests/test_main.py:22]: This test is dependent on examples folder being here, so we are skipping until we discuss testing strategy.

**Comment by rht** [comment]
https://github.com/projectmesa/mesa-examples/pull/10 has been merged, so I'm merging now.

**Comment by rht** [comment]
I deleted the temp working branch as well.

**Comment by EwoutH** [comment]
It’s a shame all git history was lost in this move. That wasn’t necessary at all. There are way more elegant ways to approach this using git. `git-filter` for example.

So much historical context lost… please never do this this way again.

**Pull request merged**